app/lib/server/server.go

```diff
@@ -319,10 +324,10 @@ func entityListToPublics(el *sda.Roster) []abstract.Point {
 	return publics
 }
 
-func isPublicIP(ip string) bool {
+func isPublicIP(ip network.Address) bool {
```

Comment:
I propose to move this to `network.Address`, as it is useful in general, no? But then we'd also have to add missing blocks - http://www.tutorialspoint.com/ipv4/ipv4_quick_guide.htm